lib/msal-core/docs/FAQ.md

```diff
@@ -379,6 +387,17 @@ You can read more about this behavior [here](https://docs.microsoft.com/azure/ac
 
 MSAL.js will only process tokens which it originally requested. If your flow requires that you send a user a link they can use to sign up, you will need to ensure that the link points to your app, not the B2C service directly. An example flow can be seen in the [working with B2C](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/working-with-b2c.md) doc.
 
+## I used to be able to get an access token when using acquireTokenSilent with clientId as scope, now I can't. Why?
```

Comment:
This can now be removed since the section above covers it. Or just link to the above if you feel like keeping it.

Reply:
my bad. done!